docs/BR.md

```diff
@@ -2030,136 +2806,62 @@ If the signing key is P-521, the signature MUST use ECDSA with SHA-512. When enc
 
 ### 7.1.4 Name Forms
 
-#### 7.1.4.1 Name Encoding
-
-Prior to 2020-09-30, the content of the Certificate Issuer Distinguished Name field MUST match the Subject DN of the Issuing CA to support Name chaining as specified in RFC 5280, Section 4.1.2.4.
-
-Effective 2020-09-30, the following requirements SHOULD be met by all newly-issued Subordinate CA Certificates that are not used to issue TLS certificates, as defined in [Section 7.1.2.2](#7122-subordinate-ca-certificate), and MUST be met for all other Certificates, regardless of whether the Certificate is a CA Certificate or a Subscriber Certificate.
-
-For every valid Certification Path (as defined by RFC 5280, Section 6):
-
-* For each Certificate in the Certification Path, the encoded content of the Issuer Distinguished Name field of a Certificate SHALL be byte-for-byte identical with the encoded form of the Subject Distinguished Name field of the Issuing CA certificate.
-* For each CA Certificate in the Certification Path, the encoded content of the Subject Distinguished Name field of a Certificate SHALL be byte-for-byte identical among all Certificates whose Subject Distinguished Names can be compared as equal according to RFC 5280, Section 7.1, and including expired and revoked Certificates.
-
-#### 7.1.4.2 Subject Information - Subscriber Certificates
-
-By issuing the Certificate, the CA represents that it followed the procedure set forth in its Certificate Policy and/or Certification Practice Statement to verify that, as of the Certificate's issuance date, all of the Subject Information was accurate. CAs SHALL NOT include a Domain Name or IP Address in a Subject attribute except as specified in [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) or [Section 3.2.2.5](#3225-authentication-for-an-ip-address).
+This section details encoding rules that apply to all Certificates issued by a CA. Further restrictions may be specified within [Section 7.1.2](#712-certificate-content-and-extensions), but these restrictions do not supersede these requirements.
 
-Subject attributes MUST NOT contain only metadata such as '.', '-', and ' ' (i.e. space) characters, and/or any other indication that the value is absent, incomplete, or not applicable.
+In addition, the following requirements apply to all Subject Attributes (i.e. those attributes within the `subject` field of a `tbsCertificate`):
+  * Subject Attributes MUST NOT contain only metadata such as '.', '-', and ' ' (i.e. space) characters, and/or any other indication that the value is absent, incomplete, or not applicable.
+  * Subject Attributes MUST NOT include a Domain Name or IP Address, except as specified in [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) or [Section 3.2.2.5](#3225-authentication-for-an-ip-address)
```

Comment:
Yeah, this is one I was struggling to try to disentangle, for obvious CAs such as `SSL.com`

We have the 7.1.4.3 rules regarding accuracy of information, and this prohibition presently exists in 7.1.4.2 (scoped to Subscriber certificates). But nothing prevents a CA certificate from being used as a server certificate (for real; nothing requires CA=false in the TLS layer; this is similar to the OCSP issue, and makes sense when you think of self-signed certs)

Ostensibly, 3.2.2.4 prohibits the `O=SSL.com` form, because it restricts FQDNs appearance, but I was trying to provide a better model for language.

I'm totally open to suggestions here! Hopefully this doesn't seem unreasonable?

Reply:
I'm not sure that we need to only be concerned about the CN. I agree with you, that reduces the technical risk of automatic acceptance, but if CAs see the value in identity information being that it can influence end user decisions/behaviours, then I think we have to consider the whole DN, right? This was the same reason for the original requirement on the `organizationalUnit` - and wanting to prevent misleading end users.

For example, if I incorporated a company in (insert lax jurisdiction here) titled "Google.com", would it be valid for a CA to issue a cert to `O=google.com`? Yes, it would seem so. What if that certificate was for `O=google.com, CN=g00gle.com` - would the user be mislead into thinking that was a legitimate Google site?

Basically, doesn't the requirement to validate the domain fall out from the "no misleading information" in the existing requirements?

Reply:
Just commenting that this discussion continues in https://github.com/sleevi/cabforum-docs/pull/36#discussion_r868254964

Reply:
(Resolving this in favor of the later comment)